### PR TITLE
Enabled airdrop rules.

### PR DESCRIPTION
**Pull request by raamb**
Added rules as static content

### CODE REVIEW

File: snet-ui/AirdropRegistration/index.tsx [hunk 1]
```diff
@@ -324,6 +324,22 @@ export default function AirdropRegistration({
...
+                <Box
+                  sx={{ display: "flex", justifyContent: "center", mt: [2, 0] }}
+                >
+                  <Button
+                    variant="contained"
+                    color="secondary"
+                    onClick={onViewRules}
+                    sx={{
+                      textTransform: "capitalize",
+                      width: 170,
```
Comment (rajeevbbqq) [inline snet-ui/AirdropRegistration/index.tsx:336]: @raamb can you test the responsiveness of the button, since we are hard-coded width of it

Comment (raamb) [reply snet-ui/AirdropRegistration/index.tsx:336]: @rajeevbbqq its the same for all the other elements. Also I just brought this back from an earlier checkin

Comment (rajeevbbqq) [reply snet-ui/AirdropRegistration/index.tsx:336]: @raamb it's ok but can you check, so we don't need to revisit for style fixes.

Comment (rajeevbbqq) [reply snet-ui/AirdropRegistration/index.tsx:336]: Let's fix the responsiveness on a separate PR

**Review by rajeevbbqq — APPROVED** [review]

**Pull request merged**